renv/activate.R

```diff
@@ -0,0 +1,1220 @@
+
+local({
```

Comment:
Should this file be here?

Reply:
Good question. Do we have some guidelines about how to use `renv`?

Reply:
yes, it should be there, otherwise renv cannot bootstrap itself.